keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -474,12 +472,12 @@ def compile(
             raise ValueError("User metrics not yet supported for YOLOV8")
 
         if isinstance(box_loss, str):
-            if box_loss == "iou":
-                box_loss = YOLOV8IoULoss(reduction="sum")
+            if box_loss == "ciou":
```

Comment:
Let's also allow "iou" here so we don't break backwards compat (maybe with a comment about why it's included)